packages/p2p-v2/src/hooks/usePoiPoaStatus.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import { useGetAccountStatus } from '@deriv/api';
+
+/** A custom hook that returns the POA, POI status and if POA is required for P2P */
+const usePoiPoaStatus = () => {
```

Comment:
can we use the shared hooks (usePOI, use POA) instead of this?

Reply:
hmmm but then the p2p_poa_required isn't part of the return in the data, and i think its better to have a combined hook specifically for p2p since the data we want is a little different than both the usePOI and usePOA